From 284c978f487d08518e730c590e1c1054a4f8cdf3 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Wed, 7 Mar 2018 10:50:35 +0530
Subject: [PATCH 0324/1851] media: xilinx: vip: Correct the fourcc code for
 BGRX8 format.

commit 221ce77a247a9f66bd0fe13cfa217a730cf4b697 from
https://github.com/Xilinx/linux-xlnx.git

Correct the V4L2_PIX_FMT fourcc code for BGRX8 format
to V4L2_PIX_FMT_XBGR32.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-vip.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/xilinx/xilinx-vip.c b/drivers/media/platform/xilinx/xilinx-vip.c
index 685f58780c9e..033dc004ca78 100644
--- a/drivers/media/platform/xilinx/xilinx-vip.c
+++ b/drivers/media/platform/xilinx/xilinx-vip.c
@@ -53,7 +53,7 @@ static const struct xvip_video_format xvip_video_formats[] = {
 	{ XVIP_VF_BGRX, 8, NULL, MEDIA_BUS_FMT_RBG888_1X24,
 	  4, 32, V4L2_PIX_FMT_BGRX32, 1, 1, 1, 1 },
 	{ XVIP_VF_XRGB, 8, NULL, MEDIA_BUS_FMT_RBG888_1X24,
-	  4, 32, V4L2_PIX_FMT_XRGB32, 1, 1, 1, 1 },
+	  4, 32, V4L2_PIX_FMT_XBGR32, 1, 1, 1, 1 },
 	{ XVIP_VF_XBGR, 10, NULL, MEDIA_BUS_FMT_RBG888_1X24,
 	  4, 32, V4L2_PIX_FMT_XBGR30, 1, 1, 1, 1 },
 	{ XVIP_VF_MONO_SENSOR, 8, "mono", MEDIA_BUS_FMT_Y8_1X8,
-- 
2.31.1

